dist/shiny-server-client.js

```diff
@@ -1953,9 +1954,14 @@ exports.createFactory = function (options) {
       }
     });
 
-    var conn = new global.SockJS(url, null, {
-      protocols_whitelist: whitelist
-    });
+    var transportDebugging = options.transportDebugging && options.transportDebugging == true;
```

Comment:
Is this a simple boolean cast of `options.transportDebugging`? Is there a reason you didn't just do `options.transportDebugging == true`, or `!!options.transportDebugging`?

Other than that, LGTM. Feel free to merge when ready.